src/main/java/io/managed/services/test/prometheuswebclient/QueryResult.java

```diff
@@ -0,0 +1,37 @@
+package io.managed.services.test.prometheuswebclient;
```

Comment:
this is can be shared between the Prometheus and Observatorium clients. I don't think we need to separate classes.

Reply:
what was `ObservatoriumClient` is now `ProtmeheusWebClient` , which is used for querrying both Observatorium and Prometheus directly, only ifference is that these clients are build differently (and therefore they targets different urls, resources and provide different headers in their querries, even in case you would provide same `Query` object to both of these clients. 

**Details:**
actually there is zero difference between Prometheus and former "Observatorium" (therefore now completely refactored). 
Instead I keep only PrometheusClient and also created builder to make necessary params and building process more visible and smooth.

Reply:
:+1: